site.yml

```diff
@@ -60,3 +60,30 @@
   - { role: swarm, run_as: worker }
   - { role: contiv_network, run_as: worker }
   - { role: contiv_storage, run_as: worker }
+
+# net-master hosts correspond to cluster machines that run the master
+# logic of the contiv net services
+- hosts: net-master
+  sudo: true
+  environment: env
+  roles:
+  - { role: base }
```

Comment:
consider putting bare-metal host in both the `devtest` group and `net-master` group in `inventory` file. This way this host-group can be used for vagrant setups as well, where the `base` role has already been run and it will save some provisioning time.